portfolio/src/main/webapp/volleyball/volleyball.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Volleyball</title>
+        <link rel="stylesheet" href="volleyball.css">
+        <script src="volleyball.js"></script>
+    </head>
+    <body>
+        <div id="content">
+            <div style="text-align: center; border: 1px solid">
```

Comment:
nit: put all styles in CSS files
https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns